editor/editor_vcs_interface.cpp

```diff
@@ -54,58 +62,107 @@ void EditorVCSInterface::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_modified_files_data"), &EditorVCSInterface::get_modified_files_data);
 	ClassDB::bind_method(D_METHOD("stage_file", "file_path"), &EditorVCSInterface::stage_file);
 	ClassDB::bind_method(D_METHOD("unstage_file", "file_path"), &EditorVCSInterface::unstage_file);
+	ClassDB::bind_method(D_METHOD("discard_file", "file_path"), &EditorVCSInterface::discard_file);
 	ClassDB::bind_method(D_METHOD("commit", "msg"), &EditorVCSInterface::commit);
 	ClassDB::bind_method(D_METHOD("get_file_diff", "file_path"), &EditorVCSInterface::get_file_diff);
 	ClassDB::bind_method(D_METHOD("shut_down"), &EditorVCSInterface::shut_down);
 	ClassDB::bind_method(D_METHOD("get_project_name"), &EditorVCSInterface::get_project_name);
 	ClassDB::bind_method(D_METHOD("get_vcs_name"), &EditorVCSInterface::get_vcs_name);
+	ClassDB::bind_method(D_METHOD("get_previous_commits"), &EditorVCSInterface::get_previous_commits);
+	ClassDB::bind_method(D_METHOD("get_branch_list"), &EditorVCSInterface::get_branch_list);
+	ClassDB::bind_method(D_METHOD("checkout_branch", "branch"), &EditorVCSInterface::checkout_branch);
+	ClassDB::bind_method(D_METHOD("push"), &EditorVCSInterface::push);
+	ClassDB::bind_method(D_METHOD("pull"), &EditorVCSInterface::pull);
+	ClassDB::bind_method(D_METHOD("fetch"), &EditorVCSInterface::fetch);
+	ClassDB::bind_method(D_METHOD("set_up_credentials"), &EditorVCSInterface::set_up_credentials);
 }
 
 bool EditorVCSInterface::_initialize(String p_project_root_path) {
 
-	WARN_PRINT("Selected VCS addon does not implement an initialization function. This warning will be suppressed.")
+	WARN_PRINT("Selected VCS addon does not implement  initialization function. This warning will be suppressed.");
 	return true;
 }
 
 bool EditorVCSInterface::_is_vcs_initialized() {
-
+	WARN_PRINT("Selected VCS addon does not implement \"" + String(__FUNCTION__) + "\"function. This warning will be suppressed.")
```

Comment:
Since this exact line repeats quite a lot of times, you can easily put this line in a macro and call that macro wherever you need it